src/js/config/config.js

```diff
@@ -1,7 +1,7 @@
 var config = {
   // @@ENV gets replaced by build system
   environment: "@@ENV",
-  rootUrl: "",
+  rootUrl: "assets/",
```

Comment:
I have a mild preference for the default being production, and the override being development.